_includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md

```diff
@@ -9,7 +9,7 @@ To start feature integration, overview and install the necessary features:
 | NAME     | VERSION | REQUIRED SUB-FEATURE     |
```

Comment:
same here for the frontmatter. Please also check the files below - some of them are missing the frontmatter as well

Reply:
same here